app/controllers/concerns/idv_session.rb

```diff
@@ -40,4 +40,11 @@ def idv_session
   def idv_attempter_throttled?
     Throttler::IsThrottled.call(current_user.id, :idv_resolution)
   end
+
+  def sp_context_needed?
+    return if sp_session.present?
```

Comment:
do we want to check for `sp_session` or like an actual SP? like `sp_from_sp_session.present?`